File: multiplatform-benchmark-java-graal/src/main/java/BenchmarkString.java
public class BenchmarkString {
    void benchString(int iterations) {
        var c = 0;
        var string = "";
        for (int i = 0; i < iterations; i++) {
            c += (i / 3);
            c += 1;
            var next = c % 10;
            string += next;
        }
    }
}

File: multiplatform-benchmark-kotlin/src/commonMain/kotlin/BenchmarkString.kt
class BenchmarkString {

    fun benchString(iterations: Int) {
        var c = 0L
        var string = ""
        for (i in 0..iterations) {
            c += (i / 3)
            c += 1
            val next = c % 10
            string += next
        }
    }


}